### DISCUSSION TITLE
Netherlands: NieuweStroom

### DISCUSSION THREAD

**Discussion by ikke-zelf** [JaccoR/hass-entso-e#23] [Price Modifyer Templates]
Hi,
modified the template example to be able to specify the extra costs and VAT as used in NL at the moment for anyone who want to use it.

```
{% set s = {
    "extra_cost": 0.018,
    "voorschot_cvo": 0.0001,
    "energie_belasting": 0.04,
    "ode_belasting": 0.03,
    "VAT": 1.21,
    "VAT_discount": 1.09
}
%}
{% if now().year <= 2022 %}
    {{(current_price + s.extra_cost + s.voorschot_cvo + s.energie_belasting + s.ode_belasting) * s.VAT_discount | float}}
{% else %}
    {{(current_price + s.extra_cost + s.voorschot_cvo + s.energie_belasting + s.ode_belasting) * s.VAT | float}}
{% endif %}
```

**Comment by JaccoR** [comment]
Hey, nice to share different templates! I just opened a discussion category for templates and moved this post over there. What energy provider is this from?

**Reply by ikke-zelf** [reply]
This one is NieuweStroom.
Waiting for the ANWB for opening there own energy (any moment now) that will save another 0.018 as they don't charge a surplus

**Comment by siempie76** [comment]
@ikke-zelf @JaccoR Where can I put this template so it shows up in the selection box in the configuration? As a .py file I assume?

**Reply by JaccoR** [reply]
That feature is not in there yet. I am not sure if it will be in the shortly.

**Reply by siempie76** [reply]
@JaccoR Where can I put this then to have the pricing altered?

**Comment by ikke-zelf** [comment]
You can put it in the configuration of the ENTSO integration
settings-> integrations-> ENTSo configuration

Same window as where you enter your API key and location

**Reply by siempie76** [reply]
yes I can select the template there, but where do I store the template to have it show up in the list. Somewhere in the entsoe directory? I can't paste the contents above in the window where you can enter your api and location. Sorry for the noob questions :-)

**Reply by Motorrad-NL** [reply]
![Screenshot_20221018-140756_Home Assistant](https://user-images.githubusercontent.com/78051647/196425224-0b9c4f05-c7eb-4293-adbc-17eb602faeaf.jpg)

**Comment by siempie76** [comment]
Ah yes working now. Thanks!!

**Comment by JaccoR** [comment]
In the new release, VAT can be added as an config option. This means the template does not need the VAT anymore.

**Comment by mrBlikkie** [comment]
the price are now different every 15 min what  how to setup for that ?
